src/Tickets/Commerce/Module.php

```diff
@@ -449,20 +449,43 @@ public function supports_global_stock() {
 	 * @return string
 	 */
 	public function get_price_html( $product, $attendee = false ) {
-		return tribe( Ticket::class )->get_price_html( $product, $attendee );
+		$value = $this->get_price_value( $product );
+
+		$price_html = '<span class="tribe-tickets-price-amount amount">' . esc_html( $value->get_currency() ) . '</span>';
+
+		/**
+		 * Allow filtering of the Price HTML
+		 *
+		 * @since TBD
+		 *
+		 * @param string $price_html
+		 * @param mixed  $product
+		 * @param mixed  $attendee
+		 *
+		 */
+		return apply_filters( 'tec_tickets_commerce_ticket_price_html', $price_html, $product, $attendee );
```

Comment:
We 100% want to avoid adding anything to the Module file as aligned originally on the Tickets Commerce document we are creating.

This needs to live in the Ticket Class.

Reply:
ok, just so I understand. This method was moved here from the Ticket class because of the `@todo` here: https://github.com/the-events-calendar/event-tickets/pull/2219/files#diff-960d6af64b1cae0ab20d33a9293a9aa3ae975a624399a539da3c819f31d6089bL869.

We should keep calling the methods from Tickets, then, and just move the logic in them to Ticket_Model. Is that it?